realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -664,11 +689,15 @@ void BaseRealSenseNode::setupPublishers()
                 std::shared_ptr<FrequencyDiagnostics> frequency_diagnostics(new FrequencyDiagnostics(_fps[stream], aligned_stream_name, _serial_no));
                 _depth_aligned_image_publishers[stream] = {image_transport.advertise(aligned_image_raw.str(), 1), frequency_diagnostics};
                 _depth_aligned_info_publisher[stream] = _node_handle.advertise<sensor_msgs::CameraInfo>(aligned_camera_info.str(), 1);
+
+            addMonitoredTopic(stream, TOPIC_ALIGNED_IMAGE, aligned_image_raw.str());
+            addMonitoredTopic(stream, TOPIC_ALIGNED_INFO, aligned_camera_info.str());
             }
 
             if (stream == DEPTH && _pointcloud)
             {
                 _pointcloud_publisher = _node_handle.advertise<sensor_msgs::PointCloud2>("depth/color/points", 1);
+                addMonitoredTopic(stream, TOPIC_POINTS, "depth/color/points");
```

Comment:
Out of curiosity, I don't understand yet why passing "`depth/color/points`" resolves to the topic Realsense publishes. On my running system (with our application's customization) I don't see that string.

```
$ rostopic list|grep -i depth
/foo/depth/camera_info
/foo/depth/image_rect_raw
/foo/depth_registered/camera_info
/foo/depth_registered/image_rect_raw
/foo/depth_registered/points
/foo/extrinsics/depth_to_infra1
/baa/depth/camera_info
/baa/depth/image_rect_raw
/baa/depth_registered/camera_info
/baa/depth_registered/image_rect_raw
/baa/depth_registered/points
/baa/extrinsics/depth_to_infra1
```